Service Portal - Frontend Readmes (#2243)
island-is/island.is: libs/service-portal/core/README.md

}
```

### Global Components

Global components will always be rendered by default
These are usually utility components that prompt the user about certain things or provide other global functionality

> Example: A modal providing onboarding for unfilled user profiles

Reviewer: ```suggestion
Example: A modal providing onboarding for unfilled user contact information
```

  </>
)
```

## Service Portal Shell

> The application shell takes care of initializing and maintaining the libraries along with implementing core functionality into the Service Portal.

Reviewer: ```suggestion
The application shell takes care of initializing and maintaining the libraries along with implementing core functionality added into the Service Portal.
```

### Adding a library to the shell

> Libraries are stored in the shell's store and loaded into view, to add a libary to the shell's module list, import and it to the list defined in [modules.ts](../../../apps/service-portal/src/store/modules.ts)

Reviewer: ```suggestion
Libraries are stored in the shell's store and loaded into view, to add a libary to the shell's module list, import and add it to the list defined in [modules.ts](../../../apps/service-portal/src/store/modules.ts)
```

```typescript
// other imports...
import { myNewModule } from '@island.is/service-portal/my-new-module'

export const modules: ServicePortalModule[] = [
  // other modules...
  myNewModule,
]
```

### Declaring routes for a library

> Declaring a new route for the service portal involves a few steps

Reviewer: ```suggestion
Declaring a new route for the service portal involves a few steps:
```
